Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/validation/validation_test.go

```diff
@@ -4171,6 +4171,18 @@ func TestValidateSecurityContextConstraints(t *testing.T) {
 	negativePriority := validSCC()
 	negativePriority.Priority = &invalidPriority
 
+	requiredCapAddAndDrop := validSCC()
+	requiredCapAddAndDrop.RequiredCapabilities = api.Capabilities{
+		Add: []api.Capability{"foo"},
+		Drop: []api.Capability{"foo"},
+	}
+
+	allowedCapListedInRequiredDrop := validSCC()
+	allowedCapListedInRequiredDrop.RequiredCapabilities = api.Capabilities{
+		Drop: []api.Capability{"foo"},
+	}
+	allowedCapListedInRequiredDrop.AllowedCapabilities = []api.Capability{"foo"}
```

Comment:
I don't think we need validation for this but if someone really wants it I can add it.  If you put something in both allowed and required.add it is just redundant but it isn't invalid.  It is invalid to have something in (allowed or required.add) AND required.drop which is what we're validating.